setup.cfg

```diff
@@ -51,7 +51,8 @@ test = pytest
 collect_ignore = ['setup.py']
 
 [pydocstyle]
-ignore = D105, D107, D407, D203, D213, D413, D406, D417
+convention = google
+add-ignore = D107, D407, D417, D105
```

Comment:
Are you sure we want to ignore D105 (`Missing docstring in magic method`)? I didn't do it in my PR's.

Reply:
Good catch @fealho. I agree. We should rather fix the problem.